src/main/java/net/minecraftforge/event/ForgeEventFactory.java

```diff
@@ -748,4 +749,11 @@ public static void onCommandRegister(CommandDispatcher<CommandSource> dispatcher
         MinecraftForge.EVENT_BUS.post(evt);
         return evt;
     }
+    
+    public static EntityEmittedSoundEvent entityEmittedSound(Entity source, double x, double y, double z, SoundEvent soundIn, SoundCategory category, float volume, float pitch)
```

Comment:
To keep this consistent with the other event factory methods, I suggest this be renamed to `onEntityEmittedSound`.